python/tvm/auto_scheduler/measure.py

```diff
@@ -657,9 +692,11 @@ def timed_func():
 
         if error_no == 0:
             try:
-                # TODO(FrozenGene): Update to ndarray.non-empty.
                 args = [ndarray.empty(get_const_tuple(x.shape), x.dtype, ctx) for x in
                         build_res.args]
+                random_fill = remote.get_function("tvm.contrib.random.random_fill")
```

Comment:
Seems to me that this symbol will be missing if `USE_RANDOM` is OFF?

Reply:
I think about it for a while when I implemented it. One possible way to support both ways is:
```python
if not tvm.get_global_func("tvm.contrib.random.random_fill", True):
   args = [np.empty()...]
else
   args = [np.empty()...]
   random_fill(value)
```
However, `empty` can not make us get precise measure result. And as `USE_RANDOM` is set to be `ON` by default and we don't have any special reasons to turn it off until now (like we won't turn off `USE_GRAPHRUNTIME`), so I prefer only maintain current code implementation. Of course, we could have one debate here. But IMO, I like we expose the error in compilation (for example, miss symbol) rather than make it pass but get the result we don't like (for example we go to the way of `empty` but get not enough precise measure result).

Reply:
Since currently `USE_RANDOM` is still a config, I would say we should have such a guard to make the code base more comprehensive. Imagining someone never updates `build/config.cmake` and his `USE_RANDOM` is OFF, then he may encounter crashing without a proper message. We can for example simply throw out a warning saying "USE_RANDOM is OFF and it might lead to inaccurate measurements" to work around this issue IMO.

Reply:
Yes, we could add one message. As `random fill` is the base if we want to get precise measure result, I would like to add one assert to make sure we have this function. Prompt users you should turn on `USE_RANDOM` and not just warning to make it pass. Does it make sense to you?

Reply:
Works for me. Thanks.

Reply:
done.